indico/modules/rb/views/user/rooms.py

```diff
@@ -94,6 +94,25 @@ class WPRoomBookingSearchRoomsResults(WPRoomBookingBase):
     def __init__(self, rh, menu_item, **kwargs):
         self._menu_item = menu_item
         WPRoomBookingBase.__init__(self, rh, **kwargs)
+        self.addExtraJS('''
```

Comment:
Since it's only used in RoomBookingSearchRoomsResults.tpl, I'd just add it as a script tag in there..